app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- Kindly make sure `:confirmable` is enabled in the user model 👍🏽

Reply:
Hey @rotimiazeez, thank you for the review, I just added the confirmable to the user model.
Great review performed !